src/java/org/apache/cassandra/tools/nodetool/ClientStats.java

```diff
@@ -47,6 +50,9 @@ public class ClientStats extends NodeToolCmd
     @Option(title = "list_connections_with_client_options", name = "--client-options", description = "Lists all connections and the client options")
     private boolean clientOptions = false;
 
+    @Option(title = "list_connections_with_metadata", name = "--metadata", description = "Lists all connections and authenticator-specific metadata and mode")
```

Comment:
I would probably rename this to `--verbose`. I did something similar here 

https://github.com/apache/cassandra/commit/37acd27f2d0e79fac969e34304bff2d6641728f5#diff-d4d258cb191fc8ee25c5a08c78b56505688d7d6eb2dbea6390f15dc16dfea17d

I think what you also follow by doing this is preserving some kind of a backward compatibility so users are not suddenly put in the position the output of the nodetool would be changed. I think it is a good practice to teach them to use "verbose" flag if they want to know more details.

Reply:
makes sense, replaced `--metadata` with `--verbose` and `--verbose` will simply include all fields of the connection now.